assets/helpers/tracking/guTracking.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+
+// ----- Campaigns ----- //
+
+const campaigns : {
+  [string]: string[],
+} = {
+  baseline_test: [
+    'gdnwb_copts_memco_sandc_support_baseline_support_banner',
+    'gdnwb_copts_memco_sandc_support_baseline_support_epic',
+    'gdnwb_copts_memco_sandc_support_baseline_support_liveblog',
+    'gdnwb_copts_memco_sandc_support_baseline_support_header_become_supporter',
+    'gdnwb_copts_memco_sandc_support_baseline_support_header_subscribe',
+    'gdnwb_copts_memco_sandc_support_baseline_support_side_menu_become_supporter',
+    'gdnwb_copts_memco_sandc_support_baseline_support_side_menu_subscribe',
+  ],
+};
+
+export type Campaign = $Keys<typeof campaigns>;
+
+
+// ----- Functions ----- //
+
+// Checks if a user is in a known campaign based upon their intCmp.
+function inCampaign(campaign: Campaign, intCmp: string): boolean {
+
+  return campaigns[campaign] && campaigns[campaign].includes(intCmp);
+
+}
+
+// Retrieves the user's campaign, if known, from the intCmp.
+function getCampaign(intCmp: string): ?Campaign {
+
+  let campaignName = null;
+
+  Object.keys(campaigns).forEach((campaign) => {
```

Comment:
You should be able to use .filter() here instead of forEach and then you don't need the campaignName variable.

Reply:
Nice, updated.